PokeDemoTests/MockService.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MockService.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 15/09/22.
+//
+
+import Foundation
+import Alamofire
+@testable import PokeDemo
+
+final class MockService: ServiceRepresentable {
+    
+    var sendRequestWithJSONIsCalledIndex = 0
```

Comment:
```suggestion
    var sendRequestWithJSONIsCalledCount = 0
```

Reply:
Covered in 7d2c96535ac909ec36845ef65ed69923d4c66332